.github/workflows/tests.yml

```diff
@@ -28,6 +28,10 @@ jobs:
       run: docker build -t civiform-dev --cache-from docker.io/civiform/civiform-dev:latest ./
     - name: Run tests
       run: USE_LOCAL_CIVIFORM=true bin/run-test-ci
+    - name: jacoco-badge-generator
```

Comment:
Nit: rename this to run_junit_test_and_coverage (or something like that

Reply:
The Step "name: Run tests" - actually runs the test and creates the report.. This step uses the report to generate the badge..